pandas/core/indexing.py

```diff
@@ -1231,6 +1231,14 @@ def _convert_to_indexer(self, key, axis: int):
         is_int_index = labels.is_integer()
         is_int_positional = is_integer(key) and not is_int_index
 
+        if (
+            isinstance(key, tuple)
```

Comment:
not sure if this needs a deprecation cycle: IIUC ATM the we this the same as if it were a list?

Reply:
do we have a test that exercises that case or @DriesSchaumont can you add one

Reply:
I am not sure I understand your question, could you clarify?

Reply:
my concern is that we may currently be treating `obj.loc[(a, b, c, d, e)]` like `obj.loc[[a, b, c, d, e]]`, in which case raising here would be an API change

Reply:
Added the test in the latest commit, seems to be OK.

Reply:
just so im clear: we're changing what exception/message gets raised, not *whether* we raise?

Reply:
No, this PR **does** add a new exception in the following case: when more than 1 key is used used (`len(key) > 1`) for a series  (`ndim < 2`) and when the index of that series is not a MI (`not isinstance(labels, MultiIndex)`). This new exception is added in  `_convert_to_indexer`. Note that in for example `.loc[0,1]`, key is passed to  `_convert_to_indexer` as a tuple `(0, 1)`. 

With "using a tuple raises, while using a list does not", I meant in this PR. On master, nothing gets raised in the case described above.

I changed which type/message got raised for the other, already existing, exceptions just for consistency, but I now see that this might confuse things because this PR does two things?

Reply:
I think that makes sense, thanks.

IIUC there is no way to distinguish between `ser.loc[(1, 2)]` (which a user might do thinking it is equivalent to `ser.loc[[1, 2]]`) vs `ser.loc[1, 2]` (which is pretty clearly invalid and the target of this PR)?